documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -0,0 +1,311 @@
+# Setting Up Security for a Distributed Mode Cluster
+
+In the Distributed Mode of running a cluster, each component runs separately on one or more processes, usually distributed across multiple machines. The deployment options in this mode include:
+
+* A manual deployment in hardware or virtual machines
+* Containerized deployments of these types:
+  * A Kubernetes native application deployed using the Pravega Operator
+  * A Docker Compose application deployment
+  * A Docker Swarm based distributed deployment
+
+Regardless of the deployment option used, setting up Transport Layer Security (SSL/TLS) for as well as client authentication and authorization (`Auth` in short) are important steps towards a secure deployment. TLS enables encryption of client-server and inter-component traffic and the clients to authenticate the server. Client `auth` enables the servers to authenticate and authorize the clients. Pravega strongly recommends always enabling security for production clusters.
```

Comment:
Pravega Strongly recommends enabling security, for production clusters.

Reply:
Fixed.